engine-precompiles/src/native.rs

```diff
@@ -287,13 +308,15 @@ impl<I: IO> Precompile for ExitToNear<I> {
                 // Input slice format:
                 //      recipient_account_id (bytes) - the NEAR recipient account which will receive NEP-141 ETH tokens
 
+                let eth_connector_contract_account = get_eth_connector_contract_account(&self.io)?;
+
                 if let Ok(dest_account) = AccountId::try_from(input) {
                     (
-                        current_account_id,
+                        eth_connector_contract_account,
                         // There is no way to inject json, given the encoding of both arguments
                         // as decimal and valid account id respectively.
                         format!(
-                            r#"{{"receiver_id": "{}", "amount": "{}", "memo": null}}"#,
+                            r#"{{"receiver_id":  "{}", "amount": "{}", "memo": null}}"#,
```

Comment:
Because before that, `serde` did not exist.
Yes, it should use but `TransferCallCallArgs`.

Reply:
Yeah, I think it's fine to build the JSON directly instead of moving around the `TransferCallArgs` type. The fields are simple and `ft_transfer` interface is stable, so we shouldn't expect this to change in the future.

That said, I don't see why this line is in the diff at all. The extra space that has been added doesn't seem necessary.